### PR TITLE
61 TOTAL CHANGES to update and create localized P0/P1 content for T1 locales

### PR DESCRIPTION
**Pull request by coryphwang**
1) overview@es.yaml – Updated ES translation by adding 3rd paragraph to AMP HTML section, and changing 'AMP HTML specification' link in AMP JS section.
2) overview@id.yamnl – Added new ID translation.
3) overview@ja.yaml – Updated JA translation by localizing 'title', 'hero', and 'cta' sections, adding 3rd paragraph to AMP HTML section, and changing 'AMP HTML specification' link in AMP JS section.
4) overview@ko.yaml – Updated KO translation by adding 3rd paragraph to AMP HTML section, and changing 'AMP HTML specification' link in AMP JS section.
5) overview@pt_BR.yaml – Added new pt_BR translation.
6) overview@zh_CN.yaml – Updated zh_CN translation by localizing 'title', 'hero', and 'cta' sections, adding 3rd paragraph to AMP HTML section, and changing 'AMP HTML specification' link in AMP JS section.
7) engagement@id.md – Added new ID translation.
8) engagement@ja.md – Added new JA translation.
9) engagement@pt_BR.md – Added new pt_BR translation.
10) engagement@zh_CN.md – Added new zh_CN translation.
11) experimental@es.md – Updated ES translation with new content.
12) experimental@id.md – Updated ID translation with new content.
13) experimental@ja.md – Updated JA translation with new content.
14) experimental@ko.md – Added new KO translation.
15) experimental@pt_BR.md – Updated pt_BR translation with new content.
16) experimental@zh_CN.md – Updated zh_CN translation with new content.
17) iframes@es.md – Updated ES translation with newer content, but EN has since been updated.
18) iframes@id.md – Added new ID translation, but EN has since been updated.
19) iframes@ja.md – Added new JA translation.
20) iframes@ko.md – Updated KO translation with newer content, but EN has since been updated.
21) iframes@pt_BR.md – Added new pt_BR translation.
22) iframes@zh_CN.md – Added new zh_CN translation.
23) live_blog@es.md – Added new ES translation.
24) live_blog@id.md – Added new ID translation.
25) live_blog@ja.md – Added new JA translation.
26) live_blog@pt_BR.md – Added new pt_BR translation.
27) live_blog@zh_CN.md – Added new zh_CN translation.
28) placeholders@id.md – Added new ID translation.
29) placeholders@ja.md – Added new JA translation.
30) placeholders@ko.md – Updated KO translation with newer content, though EN will be updated soon!
31) placeholders@pt_BR.md – Added new pt_BR translation.
32) placeholders@zh_CN.md – Added new zh_CN translation.
33) responsive_amp@id.md – Updated ID translation with newer content, including video, but video caption hasn't been translated.
34) responsive_amp@ja.md – Updated JA translation with newer content, including video, but video caption hasn't been translated.
35) responsive_amp@ko.md – Updated KO translation with newer content, including video, but video caption hasn't been translated.
36) responsive_amp@pt_BR.md – Updated pt_BR translation with newer content, including video, but video caption hasn't been translated.
37) responsive_amp@zh_CN.md – Updated zh_CN translation with newer content, including video, but video caption hasn't been translated.
38) style_pages@es.md – Updated ES translation with newer content.
39) style_pages@id.md – Updated ID translation with newer content.
40) style_pages@ja.md – Updated JA translation with newer content.
41) style_pages@ko.md – Updated KO translation with newer content.
42) style_pages@pt_BR.md – Updated pt_BR translation with newer content.
43) style_pages@zh_CN.md – Updated zh_CN translation with newer content.
44) use_cases@es.md – Updated ES translation with newer content.
45) use_cases@id.md – Updated ID translation with newer content.
46) use_cases@ja.md – Updated JA translation with newer content.
47) use_cases@ko.md – Updated KO translation with newer content.
48) use_cases@pt_BR.md – Updated pt_BR translation with newer content.
49) use_cases@zh_CN – Updated zh_CN translation with newer content.
50) validate@es.md – Updated ES translation with new link for 'especificación' that was missing in the 2nd to last paragraph. The rest of the content remains the same.
51) validate@id.md – Updated ID translation with newer content.
52) validate@ja.md – Updated JA translation with newer content.
53) validate@ko.md – Updated KO translation with newer content.
54) validate@pt_BR.md – Updated pt_BR translation with newer content.
55) validate@zh_CN.md – Updated zh_CN translation with newer content.
56) messages.po (ES) - Added translated text for some references to /who/amp-ads.yaml and a few other spots, but didn't change too much.
57) messages.po (ID) – Added ID translations for any references to home.yaml, menu.yaml, about-how.yaml, overview.yaml, doc.yaml, file-a-bug.md, developer.md, documentation-bug.md, platform-bug.md, slack.md, stack-overflow.md, mailing-list.md, build.md, latest.yaml, events.yaml, list-blog.html.
58) messages.po (JA) – Added JA translations for any references to home.yaml, menu.yaml, about-how.yaml, overview.yaml, doc.yaml, file-a-bug.md, developer.md, documentation-bug.md, platform-bug.md, slack.md, stack-overflow.md, mailing-list.md, build.md, latest.yaml, events.yaml, list-blog.html.
59) messages.po (KO) – Added KO translations for any references to home.yaml, menu.yaml, about-how.yaml, overview.yaml, doc.yaml, file-a-bug.md, developer.md, documentation-bug.md, platform-bug.md, slack.md, stack-overflow.md, mailing-list.md, build.md, latest.yaml, events.yaml, list-blog.html.
60) messages.po (pt_BR) – Added pt_BR translations for any references to home.yaml, menu.yaml, about-how.yaml, overview.yaml, doc.yaml, file-a-bug.md, developer.md, documentation-bug.md, platform-bug.md, slack.md, stack-overflow.md, mailing-list.md, build.md, latest.yaml, events.yaml, list-blog.html.
61) messages.po (zh_CN) – Added zh_CN translations for any references to home.yaml, menu.yaml, about-how.yaml, overview.yaml, doc.yaml, file-a-bug.md, developer.md, documentation-bug.md, platform-bug.md, slack.md, stack-overflow.md, mailing-list.md, build.md, latest.yaml, events.yaml, list-blog.html.

### CODE REVIEW

File: content/docs/guides/iframes@es.md [hunk 1]
```diff
@@ -1,60 +1,63 @@
...
+### Requisitos de `amp-iframe`:
 
-* Debe ser al menos de **600px** o **75%** de la primera ventana de visualización lejos de la parte superior.
-* Solo puede solicitar recursos a través de HTTPS, y no deben estar en el mismo origen que el contenedor, a menos que no especifique allow-same-origin.
+* Debe tener al menos **600 píxeles** o un **75%** de la primera ventana gráfica si empezamos por el principio de la página.
+* Solo puede solicitar recursos por HTTPS, y estos no deben tener el mismo origen que el contenedor salvo que no se especifique allow-same-origin.
 
-{% call callout('Read on', type='read') %}
-Aprende más sobre [lista de especificación para <code>amp-iframe</code>](/es/docs/reference/components/amp-iframe.html).
+{% call callout('Consejo', type='read') %}
```
Comment (SidVal) [inline content/docs/guides/iframes@es.md]: "Read on" is "Leer más" 
If change this, all notes with "read on" must be changed to "Consejo" :-1:

**Comment by coryphwang** [comment]
@SidVal – I went ahead and made the change that you mentioned. Thanks again for your help!

File: content/docs/guides/analytics_amp/use_cases@es.md [hunk 2]
```diff
@@ -1,245 +1,223 @@
...
-      "on": "click",
-      "selector": "a",
-      "request": "event",
-      "vars": {
-        "eventId": "42",
-        "eventLabel": "clicked on a link"
-      }
-    }
-  }
+    "requests": {
```
Comment (bpaduch) [inline content/docs/guides/analytics_amp/use_cases@es.md]: In future PRs, please do not indent 4 spaces, only 2.

File: translations/es/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -60,7 +60,7 @@ msgstr ""
 msgid ""
 "AMP Ads can serve on both AMP and non-AMP pages so you can build your AMP Ad "
 "once and deliver a fast, memorable brand experience everywhere."
-msgstr ""
+msgstr "Los anuncios AMP pueden publicarse tanto en páginas AMP como en páginas que no lo sean, de modo que puedes crear tu anuncio AMP una vez y ofrecer una experiencia de marca memorable en todas partes."
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: ` msgid ""`
Must be empty, and translation after that line.

File: translations/es/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -103,15 +103,15 @@ msgstr "Página de Aterrizage AMP"
...
+msgstr "Descripción general de AMP"
 
 #: /views/amp-ads.html:255
 msgid "AMP Page"
 msgstr "Página AMP"
 
 #: /content/includes/lists.yaml
 msgid "AMP Roadmap"
-msgstr ""
+msgstr "Hoja de ruta de AMP"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Leave "roadmap" without translation.

File: translations/es/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -103,15 +103,15 @@ msgstr "Página de Aterrizage AMP"
 #: /content/support/faqs/supported-platforms@es.md
 #: /content/support/faqs/supported-platforms@ko.md
 msgid "AMP Overview"
-msgstr ""
+msgstr "Descripción general de AMP"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Leave without translation

File: translations/es/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -227,7 +227,7 @@ msgstr "Analítica"
 
 #: /content/includes/who.yaml
 msgid "Audio/Video"
-msgstr ""
+msgstr "Audio/Video"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Leave without

File: translations/es/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -236,7 +236,7 @@ msgstr "Mejores anuncios en la web más rápida"
 #: /content/includes/latest.yaml /content/latest/blog-ads/_blueprint.yaml
 #: /content/latest/blog/_blueprint.yaml
 msgid "Blog"
-msgstr ""
+msgstr "Blog"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -285,11 +285,11 @@ msgid ""
 "AMP Ads once and deliver a memorable brand experience everywhere. Similarly, "
 "AMP pages can be built once and distributed across a wide variety of "
 "distribution platforms simultaneously."
-msgstr ""
+msgstr "Crear el anuncio, la página de destino o el contenido web adecuados lleva su tiempo. Con los anuncios AMP, tus anuncios pueden publicarse tanto en páginas AMP como en páginas que no lo sean, de modo que puedes crear tus anuncios AMP una vez y ofrecer una experiencia de marca memorable en todas partes. Del mismo modo, las páginas AMP se crean una vez y se pueden distribuir por una amplia variedad de plataformas de distribución simultáneamente."
 
 #: /content/includes/who.yaml
 msgid "CMS"
-msgstr ""
+msgstr "CMS"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -416,7 +416,7 @@ msgstr "Contenido Dinámico - AMP componentes"
 msgid ""
 "E.g. don’t integrate things on the client side, just because that is easier, "
 "when the user experience would be better with a server side integration."
-msgstr ""
+msgstr "P. ej., no integres elementos desde el cliente solo porque sea más sencillo si realizar la integración desde el servidor mejoraría la experiencia de usuario."
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: first line empty: `msgid ""`

File: translations/es/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -428,15 +428,15 @@ msgid ""
...
+msgstr "Asegúrate de que, en el caso de que se produzcan interrupciones en AMP o de que aparezca algún problema, no afecta al resto del sitio web. Esto quiere decir que si Google AMP Cache, la API de la URL o la biblioteca fallan, la calidad del servicio debe reducirse de la manera más sutil posible. Si algo funciona con una caché de AMP, también debe funcionar sin ella."
 
 #: /content/learn/who/advertisers.yaml
 msgid ""
 "Even the most memorable creative won’t serve its purpose if the ad is slow "
 "and disruptive for a user. AMP Ads are a new way of building ads, making them"
 " as fast as AMP pages to ensure your ads do what they’re meant to do and "
 "deliver the value you’re looking for."
-msgstr ""
+msgstr "Hasta la creatividad más impactante no servirá de nada si el anuncio es lento y el usuario lo ve con interrupciones. Con los anuncios AMP descubrirás una nueva forma de crear anuncios que, además, serán tan rápidos como las páginas AMP. De este modo, puedes asegurarte de que tus anuncios sirven para el propósito para el que han sido creados y de que ofrecen el valor que buscas."
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: `msgid ""`

File: translations/es/LC_MESSAGES/messages.po [hunk 11]
```diff
@@ -662,7 +662,7 @@ msgstr ""
 
 #: /content/docs/reference/components/_blueprint.yaml
 msgid "Media"
-msgstr ""
+msgstr "Media"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 12]
```diff
@@ -812,7 +812,7 @@ msgstr "Reportar un error de plataforma en GitHub"
 
 #: /content/includes/latest.yaml /content/includes/menu.yaml
 msgid "Roadmap"
-msgstr ""
+msgstr "Hoja de ruta"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -876,7 +876,7 @@ msgstr "Canal de Slack"
 
 #: /content/docs/reference/components/_blueprint.yaml
 msgid "Social"
-msgstr ""
+msgstr "Social"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 14]
```diff
@@ -1094,7 +1094,7 @@ msgstr ""
 
 #: /content/includes/menu.yaml
 msgid "Videos"
-msgstr ""
+msgstr "Videos"
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: delete translation

File: translations/es/LC_MESSAGES/messages.po [hunk 15]
```diff
@@ -1252,17 +1252,17 @@ msgstr "o"
 msgid ""
 "“AMP was the perfect addition to our business, giving us a new level of "
 "impact for our e-commerce clients.”"
-msgstr ""
+msgstr "\"AMP ha sido el complemento perfecto para nuestra empresa, nos ha permitido conseguir un impacto mayor que nunca en nuestros clientes de comercio electrónico.\""
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: first line empty, after add translation

File: translations/es/LC_MESSAGES/messages.po [hunk 16]
```diff
@@ -1252,17 +1252,17 @@ msgstr "o"
...
 
 #: /content/includes/home.yaml
 msgid ""
 "“Across Teads’ publisher portfolio, video ads perform significantly better on"
 " AMP pages than on the traditional mobile web.”"
-msgstr ""
+msgstr "\"En el catálogo de editores de Teads, los anuncios de vídeo se ven considerablemente mejor en las páginas AMP que en los sitios web móviles tradicionales.\""
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: ` msgid ""`

File: translations/es/LC_MESSAGES/messages.po [hunk 17]
```diff
@@ -1252,17 +1252,17 @@ msgstr "o"
...
 
 #: /content/includes/home.yaml
 msgid ""
 "“If our site takes a long time to load, it doesn’t matter how great our "
 "journalism is, some people will leave the page before they see what’s there.”"
-msgstr ""
+msgstr "\"Si nuestro sitio web tarda mucho en cargarse, no sirve para nada la calidad que tengamos como periodistas, ya que la gente dejará la página antes de poder saber lo que hay en ella.\""
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: ` msgid ""`

File: translations/es/LC_MESSAGES/messages.po [hunk 18]
```diff
@@ -1040,15 +1040,15 @@ msgid ""
...
+msgstr "En estos momentos no hay eventos nuevos. Celebramos eventos durante todo el año, como por ejemplo, la Conferencia de AMP anual. Síguenos en <a href=\"\" target=\"_blank\">Facebook</a> o <a href=\"https://twitter.com/amphtml\" target=\"_blank\">Twitter</a> para conocer las últimas noticias y consultar los eventos programados."
 
 #: /content/learn/amp-design-principles.yaml
 #: /content/learn/amp-design-principles@es.yaml
 #: /content/learn/amp-design-principles@ko.yaml
 msgid ""
 "These design principles are meant to guide the ongoing design and development"
 " of AMP. They should help us make internally consistent decisions."
-msgstr ""
+msgstr "Estos principios de diseño están pensados para servir de guía para un diseño y desarrollo continuos de AMP. Nos ayudarán a realizar decisiones internas coherentes."
```
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: ` msgid ""`

File: translations/es/LC_MESSAGES/messages.po (hunk 14 above)
Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: Why? There's no harm unless it's incorrect.

File: translations/es/LC_MESSAGES/messages.po (hunk 5 above)
Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: Why?

File: translations/es/LC_MESSAGES/messages.po (hunk 10 above)
Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: Seems to build correctly as is, so leaving it.  In future @coryphwang  - keep same formatting

Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: A long time ago, I wanted to translate like that and you said that it isn't correct. :neutral_face:

Comment (bpaduch) [inline translations/es/LC_MESSAGES/messages.po]: I was under that assumption too, but I built the site locally with his changes and it works.  Try it and you'll see.

File: translations/es/LC_MESSAGES/messages.po (hunk 5 above)
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: The word "overview" does not translate to "descripción general".
In spanish, Overview is something else, but if you want to said that., oK.

File: translations/es/LC_MESSAGES/messages.po (hunk 10 above)
Comment (coryphwang) [inline translations/es/LC_MESSAGES/messages.po]: ACK. I will say that it's fairly difficult to match up the lines (word for word), but I'll try to keep the formatting the same in the future.

File: translations/es/LC_MESSAGES/messages.po (hunk 14 above)
Comment (SidVal) [inline translations/es/LC_MESSAGES/messages.po]: Translation to same word.
A long time ago I translated that words, and you rejected that changes :-1:

**Comment by SidVal** [comment]
Ah ... ok.

File: content/docs/guides/analytics_amp/use_cases@es.md (hunk 2 above)
Comment (coryphwang) [inline content/docs/guides/analytics_amp/use_cases@es.md]: ACK.

**Pull request merged**